docs/documentation/securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -349,6 +351,8 @@ responseType::
 
 Called to initialize the adapter.
 
+WARNING: if you are calling this method with `onLoad: 'login-required'` make sure to call it once because it sets authenticated state to `false` and you may end up in the redirect loop.
```

Comment:
It doesn't matter if you pass `onLoad: 'login-required'`, the `init()` method should never be called twice under any circumstances.

Reply:
Removed this condition from the warning. 😉